src/components/molecules/notification-bell/index.tsx

```diff
@@ -4,12 +4,20 @@ import BellNotiIcon from "../../fundamentals/icons/bell-noti-icon"
 
 type NotificationBellProps = {
   hasNotifications: boolean
+  onClick: () => void
 }
 
 const NotificationBell: React.FC<NotificationBellProps> = ({
   hasNotifications = false,
+  onClick = () => void 0
 }) => {
-  return <button>{hasNotifications ? <BellNotiIcon /> : <BellIcon />}</button>
+
+  return <button onClick={onClick}>
```

Comment:
we have a button in `components/fundamentals/button`, we should either use that here or refactor the notification bell to return just the icon based on the icon props. Then that component should be in a button in order to get the styling on the clickable area.